### PR TITLE
Fix crosschain staleness check

### PR DESCRIPTION
**Pull request by guibescos**
This fixes a bug where crosschain prices are considered stale because a long time passes between fetching the prices from the price service and checking the staleness. 

This bug also exists on the current pythnet deployment. I think the reason it went undetected is that the main loop is faster on pythnet because the rpc node is faster.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ ENV \
     POETRY_NO_INTERACTION=1
 
 # Install Poetry - respects $POETRY_VERSION & $POETRY_HOME
-RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python
+RUN curl -sSL https://install.python-poetry.org | python
```
Comment (guibescos) [inline Dockerfile:30]: Drive-by update dockerfile

**Review by thmzlt — APPROVED** [review]
